src/Exception/ApiException.php

```diff
@@ -9,7 +9,6 @@
 
 namespace Setle\Exception;
 
-class ApiException extends \Exception
+class ApiException extends \Exception implements \Throwable
```

Comment:
Exception is altijd Throwable

Reply:
isn't the class that extends \Exception implicitly also implements \Throwable as the class Exception already implements \Throwable ?